temoa/core/model.py

```
@@ -104,71 +110,83 @@ def __init__(M, *args, **kwargs):
         ################################################
 
         # Dev Note:  The triple-quotes UNDER the items below pop up as dox in most IDEs
-        M.processInputs = dict()
-        M.processOutputs = dict()
-        M.processLoans = dict()
-        M.activeFlow_rpsditvo = None
+        M.processInputs: t.ProcessInputsDict = dict()
```

Ruff (0.14.0)

Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)